PyNN backend

tbekolay opened the pull request:
This is a migration of https://github.com/nengo/nengo/issues/54 with some additional context and prose, fit into the template proposed in #1. That PR should be looked at first to ensure that this is the process that we want to use moving forward.

tbekolay committed:
New idea: PyNN backend

tcstewar commented:
Yes!  I was just talking to a bunch of HBP people (the ones making the Neurorobotics Platform) and we were just commenting that that would be *very* useful.  (They're really very jealous of Nengo and would way prefer that over PyNN)

tbekolay committed:
Build NEPs with Sphinx and TravisCI

tbekolay committed:
Update general documents

As a result of discussion at the last dev meeting.

tbekolay committed:
Approved: PyNN backend

tbekolay commented:
I updated this with the results of the dev meeting discussion in 47ad899. I also added some commits editing the general files in the repo, and got the project building with Sphinx and TravisCI (it will show up at https://nengo.github.io/enhancement_proposals/).

I'll merge tomorrow morning if no one has issues with these changes.
